OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -236,7 +236,14 @@ public void Activate(Order order)
 				return;
 
 			var power = Instances.Where(i => !InstanceDisabled(i))
-				.MinByOrDefault(a => (a.Self.CenterPosition - a.Self.World.Map.CenterOfCell(order.TargetLocation)).HorizontalLengthSquared);
+				.MinByOrDefault(a =>
+				{
+					if (a.Self.OccupiesSpace == null)
+						return 0;
```

Comment:
Could also be `int.MaxValue`. I don't think it would make a difference, though, the number of situations where the same support power is granted by both real and disembodied actors should be quite rare.